client/src/components/SentimentAnalysis/SentimentAnalysis.js

```diff
@@ -6,20 +6,39 @@ import {
   INPUT_LIST,
 } from '../../constants/analysis_constants';
 import { AnalysisInput, ColorBar, TermsDisplay } from './Helpers';
-import React, { useState } from 'react';
+import React, { useEffect, useState } from 'react';
 
 import { CLIENT_KEY } from '../../constants/capcha_config';
 import PropTypes from 'prop-types';
```

Comment:
Do you use proptypes in this file?

Reply:
Yeah, I put PropTypes near the end of this file